src/content/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-ui-and-tasks.mdx

```diff
@@ -100,9 +98,9 @@ Some important points about adding users:
 
   <Collapser
     id="edit-user-type"
-    title="Edit users' user type"
+    title="Edit a user's type"
   >
-Before changing your users' user type, we recommend you understand:
+Before changing the user type for a user, we recommend you understand:
```

Comment:
The double use of user in this line is a little offputting, what about something like "before changing a user's type" or "a user's account type" or something along those lines? I know it's technically "user type", but I think the reader will know what we mean even if we don't use "user type" specifically.